Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Build UI
+FROM node:10 as ui
+
+ARG GRAPHQL_ENDPOINT
+ENV GRAPHQL_ENDPOINT=${GRAPHQL_ENDPOINT}
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY ui/package*.json ./
+RUN npm install
+COPY ui /app
+
+RUN npm run build
+
+# Build API
+FROM golang:alpine AS api
+
+WORKDIR /app
+COPY api /app
+
+RUN go get -d -v ./...
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o photoview .
+
+# Copy built app to nginx environment
```

Comment:
This comment should probably be changed to `production environment` or somthing like that